ietf/doc/models.py

```diff
@@ -410,16 +409,20 @@ def friendly_state(self):
 
     def author_names(self):
         names = []
-        if self.type_id=="rfc" and self.rfcauthor_set.exists():
+        if self.type_id == "rfc" and self.rfcauthor_set.exists():
```

Comment:
Checking to see if we should make things more consistent. This returns names in favor of titlepage_names. Further down in the PR, we return titlepage_names in favor of names.